specs/ledger/hs/src/Ledger/Core.hs

```diff
@@ -100,11 +100,11 @@ newtype SlotCount = SlotCount Word64
 addSlot :: Slot -> SlotCount -> Slot
 addSlot (Slot n) (SlotCount m) = Slot $ m + n
 
--- | Subtract a slot from a slot count.
+-- | Subtract a slot count from a slot.
 --
 --   This is bounded below by 0.
-minusSlot :: SlotCount -> Slot -> Slot
```

Comment:
Awh crap! I also changed this in my PR :man_facepalming: But at least we agree :D